.github/workflows/coverage-check.yml

```diff
@@ -0,0 +1,68 @@
+name: Check Test Coverage
+
+on:
+  pull_request:
+    branches: [development]
+
+jobs:
+  test-coverage:
+    name: Check Test Coverage
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: read
+
+    steps:
+      - name: Checkout development branch
+        uses: actions/checkout@v3
+        with:
+          ref: development
+
+      - name: Setup Node
```

Comment:
Redundant step. We only need foundry to run tests so you are free to delete it

Reply:
removed